data/apps.json

```diff
@@ -0,0 +1,2102 @@
+[
```

Comment:
I am wondering why we need to add a data folder outside of the connector's folder structure

Reply:
Yes, it's the same data, but I've separated it because it will be accessed by clients, and I don't want it to be accidentally modified or interfered with. We can definitely place it inside the connector, but I thought it would be easier to access it from the root folder.

Reply:
@LarsenEric Updated the data folder location.